.github/workflows/required-checks-hack-ci.yml

```diff
@@ -0,0 +1,135 @@
+# The jobs in this workflow are necessary because they are configured as
+# required checks on PRs. Github does not currently support configuring required
+# checks per workflow. A PR that only triggers codegen-ci.yml jobs will only
+# report back with the "Codegen Tests" check. But because there is only a global
+# option for required PR checks, that PR would otherwise never satisfy the
+# "Typescript/Python/APIX Tests" checks without these noop jobs.
```

Comment:
This paragraph really helped me understand the underlying issue. A monorepo does not work well with Github required checks that are repo-wide.